src/transformers/modeling_utils.py

```diff
@@ -870,6 +947,7 @@ class PreTrainedModel(nn.Module, ModuleUtilsMixin, GenerationMixin, PushToHubMix
     base_model_prefix = ""
     main_input_name = "input_ids"
     _auto_class = None
+    _no_split_modules = None
```

Comment:
New attributes to fill on all models (for now GPT-J and T5 given as examples), that specifies the block that should not be split across devices.

Reply:
Why again should certain blocks not be split across devices?

Reply:
If you split a GPTBlock across devices, the residual connection (initial input of the block) added at the end will create a device mismatch.